.gitignore

```diff
@@ -109,3 +109,6 @@ ENV/
 # VSCode project files
 .vscode/
 *.txt
+*cigar*
```

Comment:
Please don't modify the project-level gitignore file, instead you can add whatever you want to the gitignore in your student folder as you're doing below. We'll need you to remove the changes to this file from your pull request before we can merge it.